source/simulator/solver.cc

```diff
@@ -853,6 +867,15 @@ namespace aspect
                                    solver_control_cheap,
                                    solver_control_expensive);
 
+        // We do not need the linearized_stokes_initial_guess vectory anymore, so
+        // (ab)use it for computing the initial residual when the Newton solver is used
+        // For the Newton solver we are interested in || F( x^(k-1) ) + F'( x^(k-1) )
+        // \partial x^(k-1) || to determine the correct linear tolerance, which is what
```

Comment:
\partial -> \delta

And in both lines, replace `x^(k-1)` by `x^{(k-1)}` for proper markup (3 times)

Reply:
should I also surround the equation with $?

Reply:
Oh, I didn't realize that we're in a .cc file. Forget about it in that case -- it's not going to be run through doxygen anyway, so markup doesn't matter. 

But do do the delta thing.